tests/features/finish_by_hand.feature

```diff
@@ -1,9 +1,29 @@
+
 Feature: Finish by hand
 
-***TO DO - This is still a skeleton and needs to be completed.
-After I am found to be eligible, if I click the "Finish by hand" button,
-I should be taken to the correct page.
-If I click continue on that page, I should be taken to the final page.
+# After I am found to be eligible, if I click the "Finish by hand" button,
+# I should be taken to the correct page.
+# If I click continue on that page, I should be taken to the final page.
+
+# https://stackoverflow.com/questions/247135/using-xpath-to-search-text-containing-nbsp
```

Comment:
Maybe add a description of how this link is used in this PR for future reference?

Reply:
V good point. I'll update this to be more useful

Reply:
@sperrys: The tests seem to be passing and the flow seems to me to be the right flow, so I think as soon as that note is added, this will be ready to merge!

A second thing we might discuss - what blocks a PR and what doesn't.

Reply:
Actually, this isn't requesting a change, right? So theoretically we could merge and fix this after. @niharikasingh : What do you think?

I'll still write up a proposal for that norm so it can be discussed.

Reply:
It's requesting a change, but it is not a blocking change, so I'm fine with merging this.